packages/bpk-component-nudger/README.md

```diff
@@ -47,19 +49,104 @@ class App extends Component {
 }
 ```
 
+### Configurable `BpkConfigurableNudger`
+
+```js
+import React, { Component } from 'react';
+import BpkLabel from 'bpk-component-label';
+import { BpkConfigurableNudger } from 'bpk-component-nudger';
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      value: 1,
+    };
+  }
+
+  handleChange = value => {
+    this.setState({ value });
+  };
+
+  compareValues = (a, b): number => {
```

Comment:
None of these use `this` so they probably shouldn't be instance methods and they definitely don't need to be bound to `this`

Reply:
So they shouldn't be inside the class function?

Reply:
They could either be top level functions or static methods